### ISSUE TITLE
File INput A11y issues

### ISSUE THREAD

**Issue by CillianHPE** [grommet/hpe-design-system#4879]
### Link to problem

https://hpe.sharepoint.com/:x:/r/teams/hpe-design/Shared%20Documents/HPE%20Design%20System/Accessibility/WCAG%20Rule%20Evaluation%20Template.xlsx?d=we975e19f32644f4cbd0a446f23f89277&csf=1&web=1&e=FAyzfP&nav=MTVfezQ1MzVDNDNGLTVEMDEtNEVCQi1CMDAxLUJENjYyM0VBQTI3Rn0

### Current problem

Fail the following WCAG A11y rules

- 1.1.1	Non-text content (Error icon is getting read out it is just not providing meaningful information to a screenreader user that the file is too large)
- 1.3.1	Info and Relationships (Information given through presentation (the icon displayed when a file size is too large) is not available in text nor can it be programmatically determined.)
- 1.4.6	Contrast (Enhanced)
- 1.4.13	Content on Hover or Focus (Failing because when the file name truncates there isn't a way for keyboard users to access the tooltip)
- 2.5.5	Target Size (Enhanced)
- 4.1.2	Name, Role, Value (For a screenreader users we need to make it more clear the state of what files have been added and better confirmation of when a file is removed.)

**Comment by jcfilben** [comment]
Can we create separate issues for each and put them in the Grommet repo instead of the Design System. Then tag the issues with 'accessibility' and 'bug' and add them to the accessibility project board

Don't worry about creating an issue for 1.4.6 or 2.5.5 since they are AAA rules.

For 1.4.13, don't create a new ticket just add a comment to https://github.com/grommet/grommet/issues/7552 similar to what Matt did.

**Comment by CillianHPE** [comment]
@jcfilben 

Closing this issue as I've created the issues in the Grommet repo now.

[File Input - WCAG Rule 4.1.2 Name, Role, Value](https://github.com/grommet/grommet/issues/7563)
[File Input - WCGA Rule 1.4.13 Content on Hover or Focus](https://github.com/grommet/grommet/issues/7562) (added a comment I'd like you to check)
[File Input - WCAG Rule 1.1.1 Non-text content](https://github.com/grommet/grommet/issues/7561)
